app/src/main/res/values/strings.xml

```diff
@@ -116,6 +116,8 @@
     <string name="rb_mobile">mobile</string>
     <string name="rb_email">email</string>
     <string name="verification_mode">Verification Mode</string>
+    <string name="confirmPassword">Confirm Password</string>
+    <string name="error_password">Password doesnot match.</string>
```

Comment:
this should be like
` <string name="error_password_not_match">Password doesnot match.</string>`

Reply:
Done! :+1: